File: javascript/soma_impares.js
const x = parseInt(prompt("Digite o primeiro valor"));
const y = parseInt(prompt("Digite o segundo valor"));

let soma = 0;

if (x < y) {
    for (i = x + 1; i < y; i++) {
        if (i % 2 != 0) {
            soma += i;
        }
    }
} else {
    for (i = y + 1; i < x; i++) {
        if (i % 2 != 0) {
            soma += i;
        }
    }
}

console.log("SOMA DOS IMPARES = %i", soma);

File: python/soma_impares.py
print("Digite dois numeros:")
x = int(input())
y = int(input())

menor = x
maior = y

if y < x:
    menor = y
    maior = x

soma_impares = 0

for i in range(menor + 1, maior, 1):
    if i % 2 != 0:
        soma_impares += i

print(f"SOMA DOS IMPARES = {soma_impares}")
